contracts/Comptroller/Diamond/facets/MarketFacet.sol

```diff
@@ -468,10 +485,12 @@ contract MarketFacet is IMarketFacet, FacetBase {
     }
 
     /**
-     * @notice Get the Effective liquidation Incentive for a vToken
-     * @dev This value should be used when calculating account liquidity and during liquidation checks.
-     * @param account The address of the account for which to fetch the liquidation Incentive.
-     * @param vToken The address of the vToken whose liquidation Incentive is being queried.
+     * @notice Get the Effective liquidation Incentive for or a given account and market
```

Comment:
```suggestion
     * @notice Get the Effective liquidation Incentive for a given account and market
```
Also, we should specify that the fallback mechanism is in the `getLiquidationParams`